go/integTest/shared_commands_test.go

```diff
@@ -2503,3 +2503,57 @@ func (suite *GlideTestSuite) TestDel_MultipleKeys() {
 		assert.True(suite.T(), result3.IsNil())
 	})
 }
+
+func (suite *GlideTestSuite) TestType() {
+	suite.runWithDefaultClients(func(client api.BaseClient) {
+		// Test 1: Check if the value is string
+		suite.verifyOK(client.Set(keyName, initialValue))
```

Comment:
Please generate keys using `uuid` so that each test has a unique key.

Reply:
Hi @janhavigupta007 - all keys are generated with uuid.